website/docs/docs/deploy/job-commands.md

```diff
@@ -49,9 +48,78 @@ You can add or remove as many dbt commands as necessary for every job. However,
 Use [selectors](/reference/node-selection/syntax) as a powerful way to select and execute portions of your project in a job run. For example, to run tests for one_specific_model, use the selector: `dbt test --select one_specific_model`. The job will still run if a selector doesn't match any models. 
 
 :::
-   
-    
-**Job outcome**  &mdash;  During a job run, the commands are "chained" together and executed as run steps. If one of the run steps in the chain fails, then the subsequent steps aren't executed, and the job will fail.
+
+#### Test metrics in CI <Lifecycle status="team,enterprise" />
+
+Run semantic validation on metric definition models (semantic models, metrics and saved queries) as part of your CI jobs. It supports deferral and selection of semantic nodes, ensuring code changes don’t break metrics.
+
+Semantic validation executes an explain query in the data warehouse for semantic nodes to ensure the generated SQL will execute.
+
+To test metrics in CI, navigate to your **Job setting** page:
+- Add the `dbt sl validate` command under **Commands** in the **Execution settings** section to run validation checks for all your semantic models in your CI job. There are more command use cases described in the next section. 
+- You can combine this command with state selection and deferral to run validation on any metric definitions that are downstream of model changes: `dbt sl validate --select state:modified+`.
```

Comment:
Can we note that this is the recommend path. 

"we recommend only running CI on modified semantic models in order to reduce job times. `dbt sl validate --select state:modified+`.